core/storage/suggestion/gae_models_test.py

```diff
@@ -93,6 +93,70 @@ def setUp(self) -> None:
             'reviewer_2', self.change_cmd, self.score_category,
             'exploration.exp1.thread_5', None)
 
+    def test_get_model_association_to_user(self) -> None:
+        suggestion_model = suggestion_models.GeneralSuggestionModel
+        self.assertEqual(
+            suggestion_model.get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.MULTIPLE_INSTANCES_PER_USER
+        )
+
+    def test_get_export_policy(self) -> None:
+        expected_export_policy_dict = {
+            'suggestion_type': base_models.EXPORT_POLICY.EXPORTED,
+            'target_type': base_models.EXPORT_POLICY.EXPORTED,
+            'target_id': base_models.EXPORT_POLICY.EXPORTED,
+            'target_version_at_submission':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'deleted': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'last_updated': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'status': base_models.EXPORT_POLICY.EXPORTED,
+            'author_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'final_reviewer_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'change_cmd': base_models.EXPORT_POLICY.EXPORTED,
+            'score_category': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'language_code': base_models.EXPORT_POLICY.EXPORTED,
+            'edited_by_reviewer': base_models.EXPORT_POLICY.EXPORTED
+        }
+        self.assertEqual(
+            suggestion_models.GeneralSuggestionModel.get_export_policy(),
+            expected_export_policy_dict
+        )
+
+    def test_get_in_review_translation_suggestions(self) -> None:
```

Comment:
This test is trivial. Please test the behaviour, don't just try to get coverage. See point 6 [here](https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests) (that entire section of the page is worth reading, too).

Ditto elsewhere.

Reply:
I have added further test cases wherever I could find. Is it alright now?

Reply:
I think you'll want different tests for different cases. E.g.:

- the case where no submissions in the datastore match the query
- the case where some submissions in the datastore match the query and some don't
- the case where the query is invalid

etc. Try to think about the different behaviours, and cover them systematically.

This suggestion applies elsewhere, too.

Reply:
Also, please name the corresponding tests clearly -- see point 1 [here](https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests).